pages/client/index.tsx

```diff
@@ -8,29 +13,37 @@ import { Rating } from 'primereact/rating';
 import { Toast } from 'primereact/toast';
 import { Toolbar } from 'primereact/toolbar';
 import React, { useEffect, useRef, useState } from 'react';
-import { ClientTicket, clientService } from '../../src/services/ClientService'
-
-
+import { ClientTicket, ServiceCategory, clientService } from '../../src/services/ClientService';
 import { Dropdown } from 'primereact/dropdown';
 
 const Client = () => {
     const { push } = useRouter();
+    // TODO create LoadingContext & Loading Indicator in layout
+    const [_loading, setLoading] = useState(false);
+    // TODO create customHook for serviceCategories
+    const [categories, setCategories] = useState<ServiceCategory[]>();
+    const [statuses, setStatuses] = useState<ServiceCategory[]>();
```

Comment:
Is this type supposed to be ServiceCategory, or is it supposed to be ServiceStatus?

Reply:
Thanks, it's fixed.